### PR TITLE
Added PHP-Opcache to vieuw / Memory Use / interned memory use / Key's / Hits

### PR DESCRIPTION
**Pull request by SniperVegeta**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by Jellyfrog** [comment]
Nitpicking but it should be `OPcache` and not `Opcache`

**Comment by SniperVegeta** [comment]
Sure, the only thing i dont know how to get "total" at the memorys ....

**Comment by murrant** [comment]
You'll need to agree to the CLA

**Comment by murrant** [comment]
@SniperVegeta you will need to add docs to doc/Extensions/Applications.md for this.  Especially since it has an odd configuration.  You could also have it connect directly to the web server instead of going though snmp.

**Comment by SniperVegeta** [comment]
As requested

**Comment by SniperVegeta** [comment]
@murrant Should i make an example config for centos 7 / Apache?
So its outside LibreNMS / for 127.0.0.1 only?
(Based on the standaart LibreNMS install)

**Comment by SniperVegeta** [comment]
>  You could also have it connect directly to the web server instead of going though snmp.

You got an example?
so i can look into that?

**Comment by murrant** [comment]
@SniperVegeta `$text = file_get_contents($url);`

**Comment by SniperVegeta** [comment]
tnx @murrant , But i think its to mush for me to handle atm,
I yust followed basics and vieuw what apache web stats are doing
(thay didnot do the php file_get_contents)
So i copy them, try to understand and make changes

File: doc/Extensions/Applications.md [hunk 1]
```diff
@@ -769,6 +770,24 @@ It is worth noting that this only monitors a single pool. If you want to monitor
 
 The application should be auto-discovered as described at the top of the page. If it is not, please follow the steps set out under `SNMP Extend` heading top of page.
 
+### PHP-OPCACHE
+#### SNMP Extend
+1. Copy the PHP script, opcache.php, to the desired host (needs to be surved true your website).
+In this example the www root of LibreNMS is used, change as disire
+`wget https://github.com/librenms/librenms-agent/raw/master/snmp/opcache.php -O /opt/librenms/html/`
```
Comment (laf) [inline doc/Extensions/Applications.md]: You need to rebase for this file anyway but you shouldn't save the file there, `/etc/snmp/` is the location used elsewhere.

Comment (SniperVegeta) [reply doc/Extensions/Applications.md]: @laf its a php file
Opcache needs to be prosessed true PHP-FPM u cant do it via CLI
So i dont whant to make an directory inside /etc/snmp/ for securety reasons

Comment (SniperVegeta) [reply doc/Extensions/Applications.md]: (well u can but u need extra depancies to do it)

**Comment by Jellyfrog** [comment]
@SniperVegeta do you plan on finishing this?

**Comment by SniperVegeta** [comment]
@Jellyfrog For me its done,
its served true an PHP File (and served true PHP-FPM), so user need to do someting "extra"

**Comment by Jellyfrog** [comment]
Could you rebase

**Comment by laf** [comment]
Closing due to lack of activity.

**Pull request closed without merging**